### ISSUE TITLE
Bumblebee seems not work in kernel 4.15.x of Fedora 27

### ISSUE THREAD

**Issue by eddorneles** [Bumblebee-Project/Bumblebee#948]
Bumblebee doesn't work anymore in my Fedora 27 with kernel 4.15.4-300.fc27.x86_64 version (I have also tried the 4.15.3-300.fc27.x86_64 version). The kernel 4.14.18-300.fc27.x86_64 works properly.
I have installed the  NVIDIA-Linux-x86_64-387.34 driver based on this step-by-step tutorial: [Installing Nvidia’s CUDA 9.1 on Fedora 27](https://davidwpearson.wordpress.com/2017/12/21/installing-nvidias-cuda-9-1-on-fedora-27/comment-page-1/#comment-57). The NVIDIA-Linux-x86_64-390.25 driver is already available but I'm not sure if this is supposed to work with new kernel version, that's the reason I didn't update yet.
So, I'd like to know if this is an issue with bumblebee or just a kernel incompatibility with the driver version.

**Comment by akien-mga** [comment]
Seems to be broken on Mageia 6 with kernel 4.14.20 too, while 4.14.19 and earlier worked fine.

**Comment by eddorneles** [comment]
I have updated from the driver 387.34 version to 390.25 version, but the issue stills there...
For now, I'll keep using the kernel 4.14.18-300.fc27.x86_64 version.

**Comment by gsgatlin** [comment]
I am using 4.15.4-300.fc27.x86_64 with bumblebee ok. It works fine with minecraft.  But I am using my own packages, not the ones from copr. I haven't ever had a need to use cuda.

https://www.linux.ncsu.edu/bumblebee

Don't know if that would help you. Cheers.

**Comment by RobbiNespu** [comment]
It work fine for me
![screenshot from 2018-03-02 16-33-41](https://user-images.githubusercontent.com/3077875/36890081-8b4dbe42-1e37-11e8-9818-cbbaa77383ae.png)

```
rnm@robbinespu ~] $ uname -r
4.15.4-300.fc27.x86_64
[rnm@robbinespu ~] $ cat /etc/fedora-release 
Fedora release 27 (Twenty Seven)
```

**Comment by eddorneles** [comment]
I already had those repos mentioned in [https://www.linux.ncsu.edu/bumblebee](https://www.linux.ncsu.edu/bumblebee).
I have uninstalled everything and then installed once again (using unmanaged repos accordingly to the tutorial).
I ran `'systemctl status bumblebeed.service'` after system start but the error persisted.
Finally, I stopped the bumblebee service and then started again and it was running correctly!
I ran `systemctl enable bumblebeed.service`.
Now on system start the bumblebee service runs properly!
Thank you very much guys!
